cprover_bindings/src/goto_program/location.rs

```diff
@@ -81,6 +93,68 @@ impl Location {
         Location::Loc { file, function, line, col }
     }
 
+    pub fn property_location<T, U: Into<InternedString>>(
```

Comment:
Move the constraint on `U` into the `where` clause

Reply:
Sure